src/forms/FormFieldText/FormFieldText.styled.tsx

```diff
@@ -26,16 +26,10 @@ export const StyledTextField = styled(({ fieldSize, ...rest }) => (
 		}
 	}
 
-	.MuiInputBase-multiline {
-		padding: 0px;
-	}
-
 	.MuiOutlinedInput-input {

```

Comment:
I'm checking checking how this comes through in the Form playground and it looks like the height & padding have increased post-changes. Do you reckon we should constrain these back their originals for single-line fields just so  the heights/paddings match up with other components like dropdowns/datepickers?
![image](https://github.com/simpleviewinc/sv-mosaic/assets/13272784/8c92fcbf-0d2e-4279-beff-a671e75747a6)